#!/usr/bin/env python

PACKAGE = "traversal_map"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("threshold", double_t, 0, "A threshold for determine the differences between elevation to accept as traversal", 0.7)

gen.add("map_lenght_x", double_t, 0, "A length of x coordinate of the region map", 25.6125)

gen.add("map_lenght_y", double_t, 0, "A length of y coordinate of the region map", 25.6125)

gen.add("map_resolution", double_t, 0, "A resolution of the region map", 0.1)

gen.add("base_frame", str_t, 0, "Occupancy base frame id", "/odom")

gen.add("x", double_t, 0, "Length of x coordinate of the robot", 13.0)

gen.add("y", double_t, 0, "Length of y coordinate of the robot", 8.0)

exit(gen.generate(PACKAGE, "traversal_map", "TraversalArea"))
